### ISSUE TITLE
Xrange chart: overriding colors with className handled incorrectly

### ISSUE THREAD

**Issue by Rob-DM1CM** [highcharts/highcharts#7620]
#### Expected behaviour
When trying to override the standard colors of chart bars in an xrange-type chart, one would expect the Highcharts code to overwrite the standard class name in the resulting SVG code for the bar with the user-defined className. Example:
Standard color:
`<rect x="411.5" y="125.5" width="277" height="20" rx="3" ry="3" fill="#90ed7d" class="highcharts-point highcharts-color-2 highcharts-partfill-original" stroke="gray" stroke-width="1"></rect>`
User-defined color:
`<rect x="411.5" y="125.5" width="277" height="20" rx="3" ry="3" fill="#90ed7d" class="highcharts-point highcharts-color-9 highcharts-partfill-original" stroke="gray" stroke-width="1"></rect>`

#### Actual behaviour
What actually happens is that the user-defined color is concatenated with the standard color className as in this example:
`<rect x="411.5" y="125.5" width="277" height="20" rx="3" ry="3" fill="#90ed7d" class="highcharts-point highcharts-color-2 highcharts-color-9 highcharts-partfill-original" stroke="gray" stroke-width="1"></rect>`
which is not helpful.

#### Live demo with steps to reproduce
http://jsfiddle.net/sexqphtr/ 

#### Affected browser(s)
All

**Comment by pawelfus** [comment]
Hi @Rob-DM1CM 

Thank you for reporting this issue. This is exactly how `className` should work, from our docs ( https://api.highcharts.com/highcharts/series.xrange.data.className ) : 

> className: StringSince 5.0.0
> An additional, individual class name for the data point's graphic representation.
>
> Defaults to undefined.

It looks like a separate issue but `xrange.data` is not visible in the options tree.

**Comment by TorsteinHonsi** [comment]
@Rob-DM1CM What you want to do is probably to set the `colorIndex`. There was a bug so it didn't work with xrange, but I have fixed that now. Here's a demo with the fix applied: http://jsfiddle.net/sexqphtr/2/.

PS: For CSS styling, we recommend going all the way with [Styled mode](https://www.highcharts.com/docs/chart-design-and-style/style-by-css).

> It looks like a separate issue but xrange.data is not visible in the options tree.

Fixed in the same commit.
